_posts/2016-12-23-gitbucket-4.8.markdown

```diff
@@ -24,8 +24,20 @@ The search box has been available for issues and wiki:
 
 ![Search box for issues]({{site.baseurl}}/images/gitbucket-4.8/search_box_issues.png)
 
-![Search box for wiki]({{site.baseurl}}/images/gitbucket-4.7/search_box_wiki_box.png)
+![Search box for wiki]({{site.baseurl}}/images/gitbucket-4.8/search_box_wiki.png)
 
-In addition, this release contains new WebAPI to get a single issue, performance improvenemt for the repository viewer. It also contains some improvements and bug fix. See [all closed issues in this release](https://github.com/gitbucket/gitbucket/issues?q=is%3Aclosed+milestone%3A4.8) to know details.
+## Keep pull request comments
+
+Before this release, comments on source code of the pull request had became invisible if new commits are pushed. Since this release, if new pushed commits don't update commented line, these comments are keeped on the screen.
+
+![Comment for the pull request]({{site.baseurl}}/images/gitbucket-4.8/pull_request_comment.png)
+
+In addition, this release contains:
+
+- New WebAPI to get a single issue
+- Performance improvement for the repository viewer
+- Some other improvements and bug fix
```

Comment:
~bug fix~ => bugfixes

wasn't sure of this, taken from [wiktionary](https://fr.wiktionary.org/wiki/bugfix)